apps/web/src/pages/templates/workflow/DigestMetadata.tsx

```diff
@@ -232,7 +232,8 @@ export const DigestMetadata = ({ control, index, readonly }) => {
                     <span>at</span>
                     <Controller
                       control={control}
-                      name={`steps.${index}.metadata.timed.at`}
+                      name={`steps.${index}.metadata.timed.atTime`}
+                      shouldUnregister
```

Comment:
I guess this remove data if it is not selected?

Reply:
yes it removes it from the form timed metadata object